### ISSUE TITLE
Increased strand bias?

### ISSUE THREAD

**Issue by MaribelAranda** [Clinical-Genomics/MIP#1912]
Hi,

One of our cases (number 30, a trio), seems to present increased strand bias if I am interpreting correctly IGV.

I filtered the variants to only get the coding and splicing ones and look for those affecting the index case but not the parents, as in a monogenic disease pattern. I attach the list of variants I ended up with.
[Variant_list_case30.xlsx](https://github.com/Clinical-Genomics/scout/files/6599351/Variant_list_case30.xlsx)

Many of these variants seems to present strand bias when checking IGV. Also, even if the GT call shows that no alternative allele is present in the parents, IGV shows some read for the alternative read in the parents anyway.

As an example, I send here a screenshot of a variant in ACKR3 in the index case and its mother. The GT call in this case shows that no alternative allele should be present in the mother.
Link to this variant:
https://scout.scilifelab.se/cust099/30/977508d594a12bdc6dff8cf78e68f656
![Screenshot 2021-06-04 at 16 42 22](https://user-images.githubusercontent.com/25826266/120820882-378fe000-c555-11eb-9fe6-1ad65f90f6ad.png)
![Screenshot 2021-06-04 at 16 42 48](https://user-images.githubusercontent.com/25826266/120820901-3b236700-c555-11eb-8064-66f62a51836c.png)

Link to another example in the gene NBEAL2:
https://scout.scilifelab.se/cust099/30/15fc12a61dedf12a37715f51ff2a74c2

Please, any help would be much appreciated to understand if there is something wrong with these samples. 
Thanks.

Kind regards,
Maribel

**Comment by vwirta** [comment]
As far as I understand, this particular case has not been sequenced as Clinical Genomics as it is flagged as an externally sequenced sample. Can't help checking what library prep this is.

**Comment by MaribelAranda** [comment]
Hi,
Many thanks for your quick response. 
Yes, this is a trio sequenced with NovaSeq6000, Illumina TruSeq PCR-free 350bp. And, indeed this was sequenced at NGI Stockholm, before we got to know about Clinical Genomics and Scout.

I completely agree that with this kind of filtering I can catch errors. It just caught my attention because, while doing the same kind of filtering with other trios, I cannot see that much strand bias.

The showed trio is also different in the sense that all 3 samples were sequenced at the same time, while other trios I have the index case was sequenced before, since we did not have the parents sample yet.

Maribel
